Web/CaseStudyAssignment-4/CaseStudyAssignment-4/WebContent/index.jsp

```diff
@@ -0,0 +1,22 @@
+<%@page import="com.metacube.carportal.dbconfig.ConnectionFactory"%>
+<%@page import="java.sql.Connection"%>
+<%@page import="com.metacube.carportal.db.helper.CarPortalDao"%>
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Car Portal</title>
+</head>
+<body>
+<%
+Connection connection=new ConnectionFactory().createConnection();
```

Comment:
ACTION: It is not good to create connection here. we should place this code on server side.